include/fmt/format.h

```diff
@@ -1886,22 +1897,95 @@ auto write_float(OutputIt out, const DecimalFP& fp,
   });
 }
 
+template <typename Char> class fallback_digit_grouping {
+ public:
+  explicit constexpr fallback_digit_grouping(locale_ref, bool) {}
```

Comment:
I don't think it needs to be explicit.

Reply:
Yeah, it's just copied from `digit_grouping`, done.